chordspy/chords_wifi.py

```
@@ -60,43 +118,51 @@ def process_data(self):
                 # Process your data here
                 self.data_size += len(data)
 
-                current_time = time.time()
+                current_time = time.time()  # Calculate rates every second
                 elapsed_time = current_time - self.start_time
 
                 if elapsed_time >= 1.0:
+                    # Calculate samples, packets, and bytes per second
                     sps = self.calculate_rate(self.sample_size, elapsed_time)
```

Ruff (0.11.9)

Local variable `sps` is assigned to but never used

Remove assignment to unused variable `sps`

(F841)